src/client/lobby/lobby.scss

```diff
@@ -0,0 +1,3 @@
+.lobbyHeader {
```

Comment:
Could this be changed to kebab-case please? In the JS you can use array notation `style['lobby-header']`; you may have to move back to using string concatenation, but see if `${style['lobby-header']}` works